Content.Server/Doors/Electronics/Systems/DoorElectronicsSystem.cs

```diff
@@ -0,0 +1,72 @@
+using System.Linq;
+using Content.Shared.Access.Components;
+using Content.Shared.DeviceNetwork.Components;
+using Content.Shared.Doors;
+using Content.Shared.Doors.Electronics;
+using Content.Shared.Interaction;
+using Content.Server.Doors.Electronics;
+using Robust.Server.GameObjects;
+
+namespace Content.Server.Doors.Electronics;
+
+public sealed class DoorElectronicsSystem : EntitySystem
+{
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsUpdateConfigurationMessage>(OnChangeConfiguration);
+        SubscribeLocalEvent<DoorElectronicsComponent, DoorElectronicsRefreshUiMessage>(OnRefreshUi);
+    }
+
+    public void UpdateUserInterface(EntityUid uid, DoorElectronicsComponent component)
+    {
+        var accesses = new List<string>();
+
+        if (TryComp<AccessReaderComponent>(uid, out var accessReader))
+        {
+            foreach (var accessList in accessReader.AccessLists)
+            {
+                var access = accessList.FirstOrDefault();
+                if (access == null)
+                    continue;
+                accesses.Add(access);
```

Comment:
I dislike this because this is only half implementing proper access configurations, but fixing this requires improving the UI, which needs doing anyway. So I guess that can just be done in some separate PR.